chain/chain/src/chain_update.rs

```diff
@@ -237,7 +237,7 @@ impl<'a> ChainUpdate<'a> {
             );
         }
         if let Some(state_sync_info) = state_sync_info {
-            self.chain_store_update.add_state_sync_info(state_sync_info);
+            self.chain_store_update.add_state_sync_info(*block.hash(), state_sync_info);
```

Comment:
Why not put the block hash inside of the state sync info?

Reply:
See the comment above about avoiding the DB migration

Reply:
reverted to the old code where we dont pass this block hash anymore